### DISCUSSION TITLE
Pin Article

### DISCUSSION THREAD

**Discussion by jueklu** [nunocoracao/blowfish#723] [Q&A]
Hi there,

is there a way to pin an article so that it always stays on top of all posts?
Appreciate your help,
Jürgen

**Comment by nunocoracao** [comment]
Two options:

quick and dirty - you can give it a date in the future and generate the site with a Hugo flag to display future articles (I think this is possible but I would need to check Hugo's docs). Apart from that no. 

some work - You can also override the pages where you want that to be the case and use the article shortcode to put the highlighted article there.
